components/Tag/index.tsx

```diff
@@ -87,7 +87,7 @@ export const withTagStyleProps =
         fontSize: getFontSize('md'),
         color: getColor(color, 700),
         lineHeight: getLineHeight('md'),
-        ...getTypeface(typeface, font),
+        fontFamily: getFontFamily(fontFamily, font),
```

Comment:
See my other comments. I think this will become:
```
fontFamily: getFontFamily(fontFamily),
fontWeight, // if we still want to support font-weight as a style prop.
```